plugins/chat/assets/stylesheets/common/chat-height-mixin.scss

```diff
@@ -1,23 +1,26 @@
 @mixin chat-height($inset: 0px) {
   // desktop and mobile
   // -1px is for the bottom border of the chat navbar
+
   height: calc(
     var(--chat-vh, 1vh) * 100 - var(--header-offset, 0px) -
-      var(--composer-height, 0px) - 1px - $inset
+      var(--composer-height, 0px) - var(--footer-nav-height, 0px) - 1px - $inset
   );
 
   // mobile with keyboard opened
   .keyboard-visible & {
     height: calc(
-      var(--chat-vh, 1vh) * 100 - var(--header-offset, 0px) - 1px - $inset
+      var(--chat-vh, 1vh) * 100 - var(--header-offset, 0px) -
+        var(--footer-nav-height, 0px) - 1px - $inset
     );
   }
 
   // ipad
   .footer-nav-ipad & {
     height: calc(
       var(--chat-vh, 1vh) * 100 - var(--header-offset, 0px) -
-        var(--composer-height, 0px) - 1px - $inset
+        var(--composer-height, 0px) - var(--footer-nav-height, 0px) - 1px -
+        $inset
```

Comment:
It might be helpful to abstract some of the calculations here into a SCSS variable, given that there is a lot of overlap between them. The ipad value and the original value are the same. The `.keyboard-visible` rule is also the same except for `var(--composer-height, 0px)`, so maybe we can do something like this to make it a tiny bit more readable: 

```scss
  $base-height: calc(
    var(--chat-vh, 1vh) * 100 - var(--header-offset, 0px) -
      var(--footer-nav-height, 0px) - 1px - $inset
  );

  height: calc($base-height - var(--composer-height, 0px));

  // mobile with keyboard opened
  .keyboard-visible & {
    height: calc($base-height);
  }

  // ipad
  .footer-nav-ipad & {
    height: calc($base-height - var(--composer-height, 0px));
  }
```

Reply:
Yes I hesitated to do something like this... although... I think it makes things quite clear for each platform. I might reconsider it, but at least for this PR will leave it like this.

:+1: